### ISSUE TITLE
Documentation.

### ISSUE THREAD

**Issue by amoskyalo** [amoskyalo/React-Components-Plugin#44]
We need a clear documentation for the plugin.

**Comment by Fideltodayy** [comment]
Yeah, I noticed. Also a page to showcase examples of components is highly required.
We can do this

**Comment by eo33** [comment]
@amoskyalo @Fideltodayy 
Are you still looking for someone to document the plugin? I can help out!

**Comment by amoskyalo** [comment]
Hello @eo33 . We have someone working on the documentation. You can join hands if you are still interested. @rashinarasheed

**Comment by Fideltodayy** [comment]
I wanna join on the documentation too. 
Love the product
